packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -98,14 +101,34 @@ abstract contract CallBatch is Script {
     function finish() public {
         require(bytes(domain).length != 0, "must initialize");
         require(bytes(outputFile).length != 0, "must initialize");
+        require(!complete, "already written");
         complete = true;
         writeOutput();
     }
+
+    function build(address governanceRouter) public {
```

Comment:
this always builds for recovery mode. won't work for normal chainops actions. this relates, ofc, to the comment about presently only having calls for one domain in this library. i am open to punting & refactoring this later when we need it, though=

Reply:
that is the intentional design